sync/block_listener_test.go

```diff
@@ -222,20 +222,45 @@ func TestBlockListener_DataAvailabilityBadFlow(t *testing.T) {
 	atx1.Nipst.PostProof.Challenge = poetRef[:]
 
 	// Push a block with tx1 and and atx1 into bl1.
-
 	block := types.NewExistingBlock(1, []byte(rand.RandString(8)))
 	block.Signature = signer.Sign(block.Bytes())
 	block.TxIds = append(block.TxIds, tx1.Id())
 	block.AtxIds = append(block.AtxIds, atx1.Id())
+
+	// adding block to peer1
 	err = bl1.AddBlockWithTxs(block, []*types.Transaction{}, []*types.ActivationTx{atx1})
 	require.NoError(t, err)
 
 	_, err = bl1.GetBlock(block.Id())
 	require.NoError(t, err)
-	// Sync bl2.
 
 	_, _, err = bl2.DataAvailability(block)
 	require.Error(t, err)
+
+	// create a new ATX
+	atx2 := atx(signer.PublicKey().String())
+	// create a new poet proof
+	// (amit) should we do it for the second time or can we use the former? (in real life scenario we should use different proofs from same user)
```

Comment:
for this test it doesn't Really Matter really matter

Reply:
removed both comments

Reply:
fixed in 5c6b63aabad9a2849d5c4dcc7755c9ab121a0579